packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -112,6 +112,7 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
                     <Popover
                         className={Classes.TABLE_TRUNCATED_POPOVER_TARGET}
                         constraints={constraints}
```

Comment:
you don't need this line anymore, actually would prefer removing it since it's `@deprecated`